examples/ios_app/test/fixtures/bwb.xcodeproj/rules_xcodeproj/external.xcfilelist

```diff
@@ -0,0 +1,30 @@
+$(BAZEL_EXTERNAL)/examples_ios_app_external/bundles/Utils.bundle
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/_CodeSignature/CodeResources
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/ExternalFramework
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Headers/ExternalFramework-Swift.h
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Info.plist
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64-apple-ios-simulator.swiftdoc
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64-apple-ios-simulator.swiftinterface
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64-apple-ios-simulator.swiftmodule
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64.swiftdoc
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64.swiftinterface
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/arm64.swiftmodule
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/Project/arm64-apple-ios-simulator.swiftsourceinfo
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/Project/arm64.swiftsourceinfo
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/Project/x86_64-apple-ios-simulator.swiftsourceinfo
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/Project/x86_64.swiftsourceinfo
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64-apple-ios-simulator.swiftdoc
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64-apple-ios-simulator.swiftinterface
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64-apple-ios-simulator.swiftmodule
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64.swiftdoc
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64.swiftinterface
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/ExternalFramework.swiftmodule/x86_64.swiftmodule
+$(BAZEL_EXTERNAL)/examples_ios_app_external/ExternalFramework.framework/Modules/module.modulemap
```

Comment:
I've mentioned this before (maybe only offline), but I plan to get these inner files removed from the spec, and here, in the future.